percy/snapshot.py

```diff
@@ -64,18 +114,24 @@ def percy_snapshot(driver, name, **kwargs):
       "For more information on usage of PercyScreenshot, "\
       "refer https://www.browserstack.com/docs/percy/integrate/functional-and-visual")
 
-
     try:
         # Inject the DOM serialization script
         driver.execute_script(fetch_percy_dom())
+        cookies = driver.get_cookies()
+        user_agent = driver.execute_script("return navigator.userAgent;")
```

Comment:
better to make this change as part of dom package so that we dont have to do it in all sdks + wont need another selenium call

Reply:
Update CLI.